setup.cfg

```diff
@@ -40,7 +40,7 @@ install_requires =
     scipy
     cached_property
     pyuvsim>=1.1.2
-    pyuvdata>=2.0<3.0
+    pyuvdata>=2.0<=2.2.0
```

Comment:
This should be `<2.2.0`, since the problem is from `2.2.0` onward.

Reply:
Once you make this change, you should also remove the `@pytest.mark.xfail` from the `test_cli` function in the `tests/test_simulate_cli.py` script.

Reply:
Changed to `<2.2.0`